web/src/compose_banner.ts

```diff
@@ -252,3 +257,17 @@ export function show_stream_not_subscribed_error(sub: StreamSubscription): void
     });
     append_compose_banner_to_banner_list($(new_row_html), $banner_container);
 }
+
+// Ensure the input has only one single top-level element,
+// and return a JQuery element of it.
+function parse_single_node(html_element: JQuery<Node | HTMLElement>): JQuery<Node | HTMLElement> {
+    if (html_element.length !== 1) {
+        blueslip.error("Input should contain only one top-level element.");
+        return $();
+    }
+    return html_element;
+}
```

Comment:
I don't think this is needed here since we control the input to `append_compose_banner_to_banner_list`.